static/src/javascripts/projects/common/modules/commercial/dfp/apply-creative-template.js

```diff
@@ -75,10 +75,8 @@ define([
         var creativeConfig = fetchCreativeConfig();
 
         if (creativeConfig) {
-            return Promise.all([
```

Comment:
This case is actually the most important one. Contrary to other DFP line items, our own components require additional processing (network, DOM construction, logic) before calling an ad rendered. Other modules rely on this to query the geometry of the creative, eg the spacefinder. That's why it needs to wait.

I'm just wondering, do we really need to hide the iframe? It contains a JSON payload and nothing else.